bindings/java/c_kzg_4844_jni.c

```diff
@@ -360,18 +360,19 @@ JNIEXPORT jboolean JNICALL Java_ethereum_ckzg4844_CKZG4844JNI_verifyBlobKzgProof
   Bytes48 *commitments_native = (Bytes48 *)(*env)->GetByteArrayElements(env, commitments_bytes, NULL);
   Bytes48 *proofs_native = (Bytes48 *)(*env)->GetByteArrayElements(env, proofs_bytes, NULL);
 
-  bool out;
-  C_KZG_RET ret = verify_blob_kzg_proof_batch(&out, blobs_native, commitments_native, proofs_native, count_native, settings);
+  C_KZG_RET ret = verify_blob_kzg_proof_batch(blobs_native, commitments_native, proofs_native, count_native, settings);
 
   (*env)->ReleaseByteArrayElements(env, blobs, (jbyte *)blobs_native, JNI_ABORT);
   (*env)->ReleaseByteArrayElements(env, commitments_bytes, (jbyte *)commitments_native, JNI_ABORT);
   (*env)->ReleaseByteArrayElements(env, proofs_bytes, (jbyte *)proofs_native, JNI_ABORT);
 
-  if (ret != C_KZG_OK)
-  {
-    throw_c_kzg_exception(env, ret, "There was an error in verifyBlobKzgProofBatch.");
-    return 0;
+
+  if (ret != C_KZG_OK) {
+    if (ret != C_KZG_BAD_VERIFY) {
+      throw_c_kzg_exception(env, ret, "There was an error in verifyKzgProof.");
```

Comment:
Same as above. Should be `verifyBlobKzgProofBatch`

Reply:
Fixed in 07bb381 !